org.rcpml.swt/src/org/rcpml/swt/SWTComboBridge.java

```diff
@@ -34,26 +49,72 @@ protected void construct(Composite parent) {
 		int style = SWT.BORDER | SWT.SINGLE | SWT.DROP_DOWN;
 
 		this.fCombo = constructCombo(parent, style);
+		update();
 
-		this.fCombo.addDisposeListener(new DisposeListener() {
+		initHandlers();
+		
+		DataBindingContext dbc = this.getBindingContext();
+
+//		dbc.bindValue(SWTObservables.observeSelection(this.fCombo),
+//				new ElementAttributeObservable(getNode(), "state"), null, null);
+		
+		dbc.bindValue(WidgetProperties.comboSelection().observe(fCombo), new ElementAttributeObservable(getNode(), "state"), null, null);
+		
+//		String state = getAttribute("state");
+//		if (state != null && !state.equals("")) {
+//			getController().bind(new DataBinding(
+//					new DataSourceElementContentBinding(
+//							getNode(), String.class), state));
+//		}
+		
+		String path = getAttribute("path");
+		if (path != null && !path.equals("")) {
+			getController().bind(new DataBinding(
+					new DataSourceElementAttributeBinding(getNode(), "state"), path));
+		}
+
+	}
+	
+	protected void initHandlers() {
+		Combo combo = this.fCombo;
+		
+		combo.addModifyListener(new ModifyListener() {
+			
+			public void modifyText(ModifyEvent event) {
+				String onChangeAction = getAttribute(RCPMLTagConstants.ONCHANGE_ATTR);
+				if (onChangeAction.length() > 0) {
+					IScriptContextManager manager = getController().getScriptManager();
+					IScriptingContext context;
+					try {
+						context = manager.getContextFrom(onChangeAction);
+					} catch (ScriptException e) {
+						e.printStackTrace();
+						context = null;
+					}
+					if (context != null) {
```

Comment:
Should we continue work if exception is thrown? I suggest throwing exception further or move 
```
						context.bindObject("node", getNode());
						context.executeScript(onChangeAction);
```
inside try block